Dockerfile

```diff
@@ -31,7 +32,8 @@ RUN apt-get update \
   -print0 | \
   xargs -0  -I '{}' sh -c 'minify -o "{}" "{}"'
 
-RUN CGO_ENABLED=1 GOOS=linux go build -ldflags="-linkmode 'external' -extldflags '-static'" -o ./bin/song-stitch cmd/*.go
+RUN CGO_ENABLED=1 GOOS=linux go build -ldflags="-s -w -linkmode 'external' -extldflags '-static'" -o ./bin/song-stitch cmd/*.go \
+  && upx ./bin/song-stitch
```

Comment:
We could use one of these flags, maybe `--best`?

```
  --best compress best (can be slow for big files)
  Compression tuning options:
  --lzma              try LZMA [slower but tighter than NRV]
  --brute             try all available compression methods & filters [slow]
  --ultra-brute       try even more compression variants [very slow]
  ```

Reply:
It'll give `--best` a go. I tried before with `--brute` but it was very slow (order of minutes to run).